### DISCUSSION TITLE
Japanese view count thing

### DISCUSSION THREAD

**Discussion by jtakatsuki** [Rehike/Rehike#399] [Ideas]
Today while YouTube shows a shorter message "%s 回視聴", it used to show ugly "視聴回数 %s 回".
This is very ugly, but it is accurate. It translates literally to "view times count %s times"
I think it is an interesting thing to consider.
But curious if Rehike can do it because of message comes from Innertube API directly.

**Comment by ephemeralViolette** [comment]
Better proposal: replace "チャンネル登録者数 123 人" across InnerTube responses with "123 人チャンネル登録者"  for further consistency.

![image](https://github.com/Rehike/Rehike/assets/28045018/0069138c-cd46-484a-8058-398f63e5d657)
